### DISCUSSION TITLE
DJA0231 VDSL stops working until reboot

### DISCUSSION THREAD

**Discussion by jyppy** [hack-technicolor/hack-technicolor#159] [General]
My gateway is currently running firmware version Champagne 18.1.c

- Product: *DJA0231TLS*
- ISP: *FutureBroadband*
- Countries: *Australia*
- Board: *VCNT-A*

I rooted the device using the instructions you had published

I was unable to get the WAN IPv6 interface to work initially. The ISP is using VLAN on the VDSL side but after a number of failed attempt and multiple permutations, it is now working perfectly.
`config interface 'wan6'`
`       option ifname 'wanptm0'`
`	option proto 'static'`
`	option ipv6 '1'`
`	option ip6gw '2xxx:yyyy:zzzz::1'`
`	option ip6addr '2xxx:yyyy:zzzz::2/64'`
`	option ip6prefix '2aaa:bbbb:cccc::/56'`
`config device 'wanptm0'`
`	option ifname 'ptm0'`
`	option type '8021q'`
`	option name 'wanptm0'`
`	option vid '100'`

The gateway appears to drop the VDSL network connectivity every few days. Both IPv4 and IPv6 are down whilst the VDSL service is up and connected.

trying to restart the network `/etc/init.d/network restart` or VDSL service `xdslctl stop` `xdslctl start --up` does not restore connectivity... only option is a reboot

I have noticed the below in the system log (the last entry... Still not able to free previous vectSkbDev:

`Fri Apr 23 03:11:30 2021 kern.crit kernel: [571080.200000] Set preferred PHY: Non Gfast`
`Fri Apr 23 03:11:30 2021 kern.crit kernel: [571080.200000] Line 0: VDSL2 link up, Bearer 0, us=22008, ds=63461`
`Fri Apr 23 03:11:30 2021 kern.crit kernel: [571080.200000] Line 0: VDSL2 link up, Bearer 1, us=0, ds=0`
`Fri Apr 23 03:11:30 2021 kern.warn kernel: [571080.200000] bcmxtmcfg: XTM LinkInfo. P-0, S-UP(1), Service =PTM(2)`
`Fri Apr 23 03:11:30 2021 kern.warn kernel: [571080.200000] bcmxtmcfg: ReconfSAR P-0, T-2`
`Fri Apr 23 03:11:30 2021 kern.warn kernel: [571080.573000] bcmxtmcfg: Connection UP, LinkActiveStatus=0x1, US=22008000, DS=63461000`
`Fri Apr 23 03:11:31 2021 kern.warn kernel: [571080.735000] (SetConnCfg)Logical portmask[1] Logical PortId[0]`
`Fri Apr 23 03:11:31 2021 kern.warn kernel: [571080.735000] (SetConnCfg)Created new connection`
`Fri Apr 23 03:11:31 2021 kern.warn kernel: [571080.816000] netdev path : wanptm0 -> ptm0`
`Fri Apr 23 03:11:51 2021 kern.warn kernel: [571100.634000] BcmXdslSendErrorSamples: Still not able to free previous vectSkbDev(0x08d2445100). Discarding it any!`

**Comment by LuKePicci** [comment]
@jyppy this doesn't look like a documentation issue, please use issues only for hacking documentation issues. I coverted this into a discussion to better handle this topic.
